### PR TITLE
add function test and clean useless code with some code comment nits

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
1、add function test  and validate 
2、 clean useless code 
3、nits： some code comment  and code fmt problem

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE

```

### CODE REVIEW

File: pkg/volume/cephfs/cephfs.go [hunk 1]
```diff
@@ -43,7 +44,8 @@ type cephfsPlugin struct {
 var _ volume.VolumePlugin = &cephfsPlugin{}
 
 const (
-	cephfsPluginName = "kubernetes.io/cephfs"
+	cephfsPluginName           = "kubernetes.io/cephfs"
+	linuxCephfsMountBinaryPath = "/sbin/mount.cephfs"
```
Comment (dixudx) [inline pkg/volume/cephfs/cephfs.go]: I don't think we need this binary. Currently `mount` is used.

File: pkg/volume/cephfs/cephfs.go [hunk 2]
```diff
@@ -207,10 +209,17 @@ func (cephfsVolume *cephfsMounter) GetAttributes() volume.Attributes {
 	}
 }
 
-// Checks prior to mount operations to verify that the required components (binaries, etc.)
+// CanMount checks prior to mount operations to verify that the required components (binaries, etc.)
 // to mount the volume are available on the underlying node.
 // If not, it returns an error
 func (cephfsMounter *cephfsMounter) CanMount() error {
+	exe := cephfsMounter.plugin.host.GetExec(cephfsMounter.plugin.GetPluginName())
```
Comment (dixudx) [inline pkg/volume/cephfs/cephfs.go]: No need to change this method, since we assume binary `mount` is present.

Comment (zouyee) [reply pkg/volume/cephfs/cephfs.go]: Got it, thanks!

**Pull request closed without merging**